source/aggregation-pipeline-builder.txt

```diff
@@ -68,13 +68,53 @@ Limitations
   or ``$searchMeta`` stages on a :ref:`time series collection
   <manual-timeseries-landing>`.
 
+
 .. _create-agg-pipeline:
 
-Create an Aggregation Pipeline
-------------------------------
+Create an Aggregation Pipeline : Stage View Mode
+------------------------------------------------
 
 .. include:: /includes/steps/create-agg-pipeline.rst
 
+.. _create-agg-pipeline-textview:
+
+Create an Aggregation Pipeline : Text View Mode
+-----------------------------------------------
+
+.. procedure::
+   :style: normal
+
+   .. step:: In the aggregation pipeline pane, click the 
+      :guilabel:`</> TEXT` toggle switch to enable text mode for 
+      pipeline editing.
+
+      .. figure:: /images/compass/agg-builder-textview-toggle-on.png
+         :alt: Toggle textmode on
+
+   .. step:: Enter valid aggregation syntax in the newly prompted 
+      text editor. The following syntax will limit the query results 
+      to ``4`` documents.
+
+      .. code-block:: javascript
+
+         [ { "$limit" : 4 } ]
+
+      .. figure:: /images/compass/agg-builder-textview-edit.png
+         :alt: Edit the text aggregation pipeline
+
+   .. step:: Click :guilabel:`Run` at the top right of the pipeline builder. 
+      |compass-short| returns your results in the document view.
+
+      .. figure:: /images/compass/agg-builder-textview-agg-results.png
+         :alt: Results after Running Aggregation
+
+   .. step:: **(Optional)** Click the :guilabel:`OUTPUT OPTIONS` dropdown 
```

Comment:
Per the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/capitalization/#capitalize-ui-labels-as-shown-on-the-ui), we should use initial capitalization for UI elements which appear in all caps:

> As an exception, any UI element that appears in all capitals should be initial capitalized in the documentation. It reads as shouting at the user otherwise.

Suggest making this change for other all-caps UI elements mentioned in this PR as well.